lib/DDG/Spice/IsItUp.pm

```diff
@@ -8,14 +8,17 @@ use Net::IDN::Encode qw(domain_to_ascii domain_to_unicode);
 use Net::Domain::TLD qw(tld_exists);
 use Data::Validate::IP qw(is_ipv4);
 
-triggers query_lc => qr/^((?:is\s|))(?:https?:\/\/)?([\p{Alnum}\-]+(?:\.[\p{Alnum}\-]+)*?)(?:(\.\pL{2,})|)\s(?:up|down|working|online|status)\?*$/i;
+
+triggers start => "isitup", "isitdown" , "is";
+triggers end => "up", "down", "working", "online", "status";
+
 
 spice to => 'https://isitup.org/$1.json?callback={{callback}}';
 
 spice proxy_cache_valid => "418 1d";
 
 handle matches => sub {
-
+    return unless grep {$req->query_lc}=~ /^((?:is\s|isitup\s|isitdown\s|))(?:https?:\/\/)?([\p{Alnum}\-]+(?:\.[\p{Alnum}\-]+)*?)(?:(\.\pL{2,})|)\s(?:up|down|working|online|status)\?*$/i;
```

Comment:
@gautamkrishnar no need for `grep`, you can use `m/$regex/`

You also need to change the `handle` to `remainder_lc` or `query_lc`. Depending which you choose you'll have to adjust this regex.